ttnn/cpp/ttnn/operations/data_movement/tilize/tilize.hpp

```diff
@@ -35,7 +36,6 @@ struct ExecuteTilize {
         const std::optional<MemoryConfig> &memory_config = std::nullopt,
         std::optional<DataType> output_dtype = std::nullopt,
         bool use_multicore = false) {
-        constexpr uint8_t DefaultQueueId = 0;
```

Comment:
how many DefaultQueueId did you get?

Reply:
I don't know, too many!

But I am guilty of this one myself. 
I added the constant at one place. It got copied all over the world extremely fast.
But lesson learned.